src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$main-color: #00acdc;
+$main-width: 166px;
+$font-weigth-bold: 700;
```

Comment:
There is a typo in the variable name `$font-weigth-bold`. It should be `$font-weight-bold`. Correct this to avoid any issues in your stylesheets.